index-2.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta
+      name="description"
+      content="this is my professional portfolio which contains my education and my experience and projects"
+    />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Kerlos's Portfolio</title>
+    <meta name="description" content="" />
+    <!-- Render all elements normally -->
+    <link rel="stylesheet" href="./css/normalize.css" />
+    <!-- Css file  -->
+    <link rel="stylesheet" href="./css/master-2.css" />
+
+    <!-- Font awsome library -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <!-- Google Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700;800;900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <!-- header -->
+    <header>
+      <p>
+        <a class="food-1" href="./index.html">
+          <img class="food" src="./picture/bird2.jpg" alt="Bird House"
+        /></a>
+        <a href="./index.html" class="logo"> Bird House</a>
+      </p>
+      <nav class="menu">
+        <div class="nav-links-og">
+          <a class="nav-transition-og" href="./index-2.html">About</a>
```

Comment:
- [optional] great job following the original design you can set the about page to show home in the nav when you are in it instead of still showing about